tests/foreman/cli/test_computeresource_rhev.py

```diff
@@ -267,7 +272,7 @@ def test_negative_add_image_rhev_with_invalid_name(self):
         """
 
     @tier2
-    @skip_if_bug_open('bugzilla', 1278917)
+    @pytest.mark.deselect(reason="BZ:1278917")
     @stubbed()
```

Comment:
It's stubbed, meaning it would always be deselected anyway (never be selected in first place? don't know what pytest does internally)

Reply:
We run tests in CI using `-m not stubbed` this will `deselect` all the stubbed test cases before pytest can even catch the `.deselect` marker.

But I think we can keep the marker here, so our script can catch the status of the BZ (the script will collect everything including stubs) and notify changes that will be useful to update the stubbed for when it is automated.